js/components/controlBar.js

```diff
@@ -224,16 +215,24 @@ var ControlBar = React.createClass({
 
     var playheadTime = isFinite(parseInt(this.props.currentPlayhead)) ? Utils.formatSeconds(parseInt(this.props.currentPlayhead)) : null;
     var isLiveStream = this.props.authorization.streams[0].is_live_stream;
-    var videoQualityPopover = this.state.showVideoQualityPopover ? <VideoQualityPopover {...this.props} togglePopoverAction={this.toggleQualityPopover}/> : null;
     var durationSetting = {color: this.props.skinConfig.controlBar.iconStyle.inactive.color};
     var timeShift = this.props.currentPlayhead - this.props.duration;
-    // checking timeShift < 2 seconds (not == 0) as processing of the click after we rewinded and then went live may take some time
-    var playheadTimeContent = isLiveStream ? (Math.abs(timeShift) < 2 ? null : Utils.formatSeconds(timeShift)) : playheadTime;
+    // checking timeShift < 1 seconds (not == 0) as processing of the click after we rewinded and then went live may take some time
+    var isLiveNow = Math.abs(timeShift) < 1;
```

Comment:
@mityaha, i thought if we get farther than one second then we can as well just show the real time difference? I do not insist though.